_data/y2019/sessions.yaml

```
@@ -40,9 +40,9 @@
   github: kazukazuinaina
   twitter: get_me_power
   abstract: >-
-    In this presentation, I will talk about track to become Vim plugin
-    developer from Vim's User as a student and changing  my heart about Vim. I
-    will suggest you new sense of values about Vim.
+    In this presentation, I would like to talk about the trajectory from my Vim
```

```suggestion
    In this presentation, I would like to talk about the history of my Vim
```
でどうでしょうか？

https://ejje.weblio.jp/content/trajectory
辞書を見ると物理的な軌道だったので、ここで言いたいのは時間経過のあれだよなあ、という推測に基づきます

---

チェックありがとうございます！